e2e-tests/testdata/echo/go.mod

```diff
@@ -2,7 +2,6 @@ module encore.app
 
 go 1.18
 
-require "encore.dev" v1.1.0
-
-replace encore.dev => ../../../../../runtime
+require "encore.dev" v1.9.3
 
+replace encore.dev => ../../../runtime
```

Comment:
This is just unrelated cleanup, the replace directive needed to be updated since we moved the `e2e-tests` directory